src/without_ansi.rs

```diff
@@ -0,0 +1,41 @@
+use core::fmt::Display;
+
+use crate::AnsiCodeIterator;
+
+/// A wrapper struct that implements [`core::fmt::Display`], only displaying non-ansi parts.
+pub struct WithoutAnsi<'a> {
```

Comment:
I don't want this in a separate module. Please move it just above the `AnsiCodeIterator` definition.